pom.xml

```diff
@@ -40,7 +40,7 @@
 	<description>SDE for CatenaX</description>
 
 	<properties>
-		<java.version>18</java.version>
```

Comment:
What is the reason behind this downgrade? Can you please add an explanation to the description.

Reply:
I believe there are two reasons:
1. Base image shall be considered as safe, especially from IP point of view
2. 17 is a LTS version of JDK. 18 is moved to archive